tests/Browser/BookRoomsTest.php

```diff
@@ -199,6 +200,7 @@ public function testPromptedToDownloadFormAndAttach()
                 ->type('#attendees', 3)
                 ->pause(250);
 
+            # Covers "Attach Reference Files #185"
             $browser->mouseover('#submit-booking-create')
                 ->check("#bake-sale-checkbox")
                 ->assertSee('Please remember to attach a filled out "Bake Sale Form" to the document section')
```

Comment:
This test needs to be expanded to cover the file attachment process

Reply:
I'll also note that the `testPromptedToDownloadFormAndAttach()` test seems to cover this functionality already